RedHat-UX/red-hat-design-system
# docs(avatar): update best practices

## What I did

1. Used Do/Don't format for avatar's "Best practices" section


## Testing Instructions

1. Check avatar's ["Best practices" section](https://deploy-preview-2098--red-hat-design-system.netlify.app/elements/avatar/guidelines/#best-practices) in DP

## Notes to Reviewers
